src/simsearch.cc

```diff
@@ -22,6 +22,13 @@
 #include "simsearch.h"
 #include "utils/fileops.hpp"
 #include "utils/utils.hpp"
+#ifdef USE_FAISS
+#include "faiss/IndexIVF.h"
+#include "faiss/OnDiskInvertedLists.h"
+#include "faiss/IndexPreTransform.h"
+#include "faiss/index_factory.h"
+#include "faiss/gpu/GpuCloner.h"
```

Comment:
Sorry, I think this needs a separate `#ifdef USE_GPU_FAISS` as well :)
```
[ 90%] Building CXX object src/CMakeFiles/ddetect.dir/simsearch.cc.o
In file included from /opt/deepdetect/build/faiss/src/faiss/gpu/GpuIndex.h:12:0,
                 from /opt/deepdetect/build/faiss/src/faiss/gpu/GpuCloner.h:15,
                 from /opt/deepdetect/src/simsearch.cc:30:
/opt/deepdetect/build/faiss/src/faiss/gpu/utils/MemorySpace.h:11:18: fatal error: cuda.h: No such file or directory
```

Reply:
wow, sorry for that, and thanks a lot again !

Reply:
No worries, thanks for the quick fix!